### PR TITLE
feat: return Mesh object

### PR DESCRIPTION
**Pull request by william-silversmith**
The Mesh object is already transformed into numpy arrays in the
correct data types and spatial relations. This makes things
easier to use.

### CODE REVIEW

**Comment by william-silversmith** [comment]
Resolves #3

**Comment by jingpengw** [comment]
@william-silversmith I think this design is better for general usage. 
adding a function to export neuroglancer binary format for `Mesh` class would be better for us?

File: zmesh.pyx [hunk 1]
```diff
@@ -57,9 +76,22 @@ class Mesher:
```
Comment (jingpengw) [inline zmesh.pyx]: I think we are using simplification error of 40, not 8?
default simplification_factor should be higher too?

File: zmesh.pyx [hunk 2]
```diff
@@ -23,6 +23,25 @@ cdef extern from "cMesher.hpp":
...
+
+  """
+  def __init__(self, vertices, faces, normals):
+    self.vertices = vertices
+    self.faces = faces
+    self.normals = normals
+
+  def __len__(self):
+    return self.vertices.shape[0]
+
```
Comment (jingpengw) [inline zmesh.pyx]: adding a function to get neuroglancer binary representation?

**Comment by manuel-castro** [comment]
Looks good to me

**Pull request merged**